modules/storages/lib/open_project/storages/engine.rb

```diff
@@ -142,14 +142,14 @@ def self.permissions
       # condition ("if:"), caption and icon.
       menu :admin_menu,
            :storages_admin_settings,
-           { controller: '/storages/admin/storages', action: :index },
+           { controller: "/storages/admin/storages", action: :index },
```

Comment:
This is probably your editor changing this manually. And the question whether to use single or double quotes can lead to heated discussions. But I would find it best if these just weren't changed at all in all lines that haven't changed semantically. Is there a way to prevent your editor from doing this?

Reply:
It is not my editor. Usually I apply `rubocop -a` to fix style before pushing. That is the outcome of it. I see your point. The PR size is huge already and it adds more to it. My bad. In the future I'll try to keep separate and not include lines that semantically has not changed.